src/main/webapp/app/admin/metrics/metrics.service.ts

```diff
@@ -11,13 +11,14 @@ export class JhiMetricsService {
 
     // get the Registry's metrics
     getMetrics(): Observable<any> {
+        // There is still a "-" in "/management/jhi-metrics" as the JHipster Registry don't use v3.0.0+ of the JHipster dependencies
```

Comment:
So you need to do the upgrade ! This is not a good enough solution, It will fail when switching the view to microservice metrics.

Reply:
I agree that it is not a pretty fix. 

However, we were discussing that issue with @jdubois, and we were not sure how easy it is to upgrade the Registry to support v3.0.0 of the JHipster dependencies, so we wondered if we should focus on upgrading it or working on something else...

Reply:
It's not hard and it is very important in order to have the same Spring boot, Spring cloud version as the main gen. @pascalgrimmaud and I can help you as we are the maintainers of the jhipster-registry. But we just don't have any time to do it, also we don't use the registry as part of our day job anymore...

Reply:
I'll take a look !

Reply:
OK let's give a try at the upgrade, but I'm not sure we have the bandwidth to do this before JHipster v6. Those parts of Spring Cloud shouldn't have moved, as the Netflix projects we rely on haven't had any upgrade.

Reply:
I would not count on it too much. Basically what often changes is the Spring cloud code and not the Eureka code.

Reply:
Should be done by #340